### ISSUE TITLE
Kick commands

### ISSUE THREAD

**Issue by wherron01** [BORG-Groupme/BORG#3]
Add commands, with permissions, to kick members, kick whole roles, purge inactive members, and kick oneself

**Comment by xxc3nsoredxx** [comment]
* Kicking individual users
* Kicking batches of users:
  * list of names
  * is a lower privilege level than you
  * is added by a specific user
  * is inactive
  * is new
  * is self (requires no privileges)
* Re-add batch of users:
  * is kicked by a specific user (undo vandalism)

**Comment by wherron01** [comment]
the option to kick everyone of a lower privilege should not be just freely available though.  i don't think most people would want low-tier mods given ban-all-regulars powers

**Comment by xxc3nsoredxx** [comment]
No, you're right. It's meant to illustrate that you can't kick a user above your level.
